tests/test.sh

```diff
@@ -16,14 +16,26 @@
 
 set -xe
 
-[[ $# -eq 2 ]] || (echo "missing input trace file and/or kernels directory" && exit -1)
+[[ $# -eq 3 ]] || (echo "missing input trace file, kernels directory, and/or test type (cl|spv)" && exit -1)
```

Comment:
I would prefer for it not to be configurable, and just take the single trace, and extract everything out of it.